src/core/vdom/vnode.js

```diff
@@ -19,6 +19,7 @@ export default class VNode {
   isComment: boolean; // empty comment placeholder?
   isCloned: boolean; // is a cloned node?
   isOnce: boolean; // is a v-once node?
+  isAsyncPlaceholder: boolean; // placeholder node for async component
```

Comment:
Having experimented with number flags, memory consumption only drops 1%, yet manipulating flags is much more verbose than before. I didn't see much value here.